scanner/e2etests/helmchart/templates/db-deployment.yaml

```diff
@@ -33,13 +33,15 @@ spec:
           value: "scram-sha-256"
         - name: POSTGRES_PASSWORD_FILE
           value: "/run/secrets/stackrox.io/secrets/password"
+        - name: SCANNER_DB_INIT_BUNDLE_ENABLED
+          value: "true"
         resources:
           limits:
-            cpu: 2
-            memory: 4Gi
+            cpu: 4
+            memory: 10Gi
```

Comment:
is this made up or is this actually required?

Reply:
No, I don't think that's needed. I bumped Scanner's limit because we are mounting `/tmp` in memory -- to avoid going out of disk when pulling layers to index, this was just me copying and pasting things around.

Reply:
Oh, I bumped the CPU in the init to speed up the dump load; I'd like to know if that limit bumps up the limit of the actual DB, too.